Orange/canvas/application/errorreporting.py

```diff
@@ -38,6 +38,19 @@ class OWWidget: pass
 log = logging.getLogger()
 
 
+def get_developer_id():
+    """Check if developer provided his name for Sentry reports."""
+    try:
+        from Orange.misc.environ import data_dir_base
+        fname = os.path.join(data_dir_base(), 'Orange', 'developer_id.txt')
+        if os.path.exists(fname):
+            with open(fname, 'r') as f:
+                username = f.readline().strip()
+                return username
+    except Exception:
+        return ''
```

Comment:
This methods can return three different things.

username, if file was found and in correct format
'', if there was some error (something went wrong when opening/reading file)
None, it the file does not exist.

Could we merge the last two cases?